rumqttd/rumqttd-bridge1-1.toml

```diff
@@ -0,0 +1,89 @@
+id = 0
```

Comment:
we can remove all this configuration files from here. we can maybe talk about different config settings in rumqtt docs